pkgs/top-level/all-packages.nix

```diff
@@ -13633,7 +13635,10 @@ in
 
   mtpfs = callPackage ../tools/filesystems/mtpfs { };
 
-  mtxclient = callPackage ../development/libraries/mtxclient { };
+  mtxclient = callPackage ../development/libraries/mtxclient {
+    # https://github.com/NixOS/nixpkgs/pull/85922#issuecomment-619287177
+    stdenv = if stdenv.isDarwin then llvmPackages_9.stdenv else stdenv;
```

Comment:
Given the comment above, this doesn't make sense, we are using clang-7 and upstream tests with clang-6. Anyway, if this doesn't help to fix the build, we probably don't need to take in this trivial change.

Reply:
I got the impression that using this stdenv does help, but I'll put my mind to it no more. I'm putting a single commit for both mtxclient and nheko with a link to that comment by upstream.